web/js/bundle.js

```diff
@@ -55,7 +55,13 @@ var ApiClient = /*#__PURE__*/function () {
      * @type {Array.<String>}
      */
 
-    this.authentications = {};
+    this.authentications = {
+      'cookieAuth': {
+        type: 'apiKey',
+        'in': 'query',
+        name: 'SESSION_ID'
+      }
+    };
```

Comment:
Idk why this was generated

Reply:
changes to swagger were merged into master and the bundle.js was not updated yet. i have same changes in my branch.

Reply:
#113